vmdb/app/models/metric/ci_mixin/capture/openstack_base.rb

```diff
@@ -94,7 +87,8 @@ def meter_names(metric_capture_module)
 
   def find_meter_counters(metric_capture_module, resource_filter, metadata_filter, log_header)
     counters = list_resource_meters(resource_filter, log_header) + list_metadata_meters(metadata_filter, log_header)
-    counters.select { |c| meter_names(metric_capture_module).include?(c["name"]) }
+    # Select only allowed counters, with unique names
+    counters.select { |c| meter_names(metric_capture_module).include?(c["name"]) }.uniq { |x| x['name'] }
```

Comment:
Wow, so it's possible to have resource meters and metadata meters with the same name? 

Is there any deeper implications here?  Or, just an unfortunate overlap?